{"message":"Something failed during start up","stack":"Error: Something failed during start up\n    at Object.<anonymous> (/Users/jayvelasco/Documents/Work ITems/foreach/node course/vidly/index.js:45:7)\n    at Module._compile (internal/modules/cjs/loader.js:678:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)\n    at Module.load (internal/modules/cjs/loader.js:589:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:520:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:719:10)\n    at startup (internal/bootstrap/node.js:228:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:576:3)","level":"error","timestamp":"2018-07-16T16:26:46.863Z"}
{"message":"something failed miserably","stack":"Error: something failed miserably\n    at Object.<anonymous> (/Users/jayvelasco/Documents/Work ITems/foreach/node course/vidly/index.js:50:26)\n    at Module._compile (internal/modules/cjs/loader.js:678:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)\n    at Module.load (internal/modules/cjs/loader.js:589:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:520:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:719:10)\n    at startup (internal/bootstrap/node.js:228:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:576:3)","level":"error","timestamp":"2018-07-16T16:33:45.071Z"}
